module/genericsUtils.ts

```diff
@@ -70,6 +76,34 @@ const startAPI = async (apiKey: string, modelKey: string, modelInputs: object, s
     return callID
 }
 
+const feedbackAPI = async (apiKey: string, callID: string, feedback: object): Promise<any> => {
+    const url = endpoint.concat("feedback/")
```

Comment:
something I forgot to mention in middleware is a small nitpick: should we do this as feedback/v1 instead? I guess we control it and could be introducing unnecessary slowness in our dev process by doing these nitpicks, but it's a thought.